docs/lib/datastore/fragments/android/getting-started/configuration.md

```diff
@@ -2,12 +2,27 @@
 
 In your app's `onCreate` (or similar lifecycle callback), initialize Amplify:
```

Comment:
"Similar" is dubious advice to me - can we be prescriptive here?

Reply:
fixed